fe/fe-core/src/main/java/org/apache/doris/load/loadv2/MysqlLoadManager.java

```diff
@@ -91,10 +143,51 @@ public LoadJobRowResult executeMySqlLoadJobFromStmt(ConnectContext context, Load
                     loadResult.incSkipped(result.get("NumberFilteredRows").getAsInt());
                 }
             }
+        } catch (Throwable t) {
+            LOG.error("Execute mysql load failed", t);
+            // drain the data from client conn util empty packet received, otherwise the connection will be reset
```

Comment:
Do we have the same problem in normal mysql connection?

Reply:
Only mysql load will continuous send bytes. Other sql will send query at beginning